### PR TITLE
Update config file.

### PR DESCRIPTION
**Pull request by vext01**
Tidy up and I'd like to include our pre/post execution commands in our in-tree config. Would anyone object?

CC @snim2

### CODE REVIEW

**Comment by snim2** [comment]
lgtm

File: warmup.krun [hunk 1]
```diff
@@ -126,16 +126,49 @@ SKIP=[
...
+        "sudo /etc/rc.d/smtpd start",
+    ]
+else:
+    assert False
+
+
+# Copy off results after each execution -- soft-dev specific!
+POST_EXECUTION_CMDS.append(
+    "tar czf - ${KRUN_RESULTS_FILE} ${KRUN_LOG_FILE} | ssh -i id_rsa "
+    "vext01@bencher2.soft-dev.org 'cat > research/krun_results/bencher3.tgz'")
```
Comment (ltratt) [inline warmup.krun]: Why don't we make the `bencher3` bit be the output from the `hostname` command? That way we can use the same config file for all machines (I think...).

File: warmup.krun [hunk 2]
```diff
@@ -126,16 +126,49 @@ SKIP=[
...
+
+assert len(PRE_EXECUTION_CMDS) == 2
```
Comment (ltratt) [inline warmup.krun]: I'm not sure I see the point of this assertion (or the one following). Remove?

Comment (vext01) [inline warmup.krun]: I want to be _really_ sure we have the commands in place, as if they are not, we wasted time running as our results will be invalid.

File: warmup.krun (hunk 1 above)
Comment (vext01) [inline warmup.krun]: Oops, I was supposed to have done that! Good spot.

**Comment by vext01** [comment]
Try this.

**Comment by ltratt** [comment]
Works for me! Squash and then I'll merge?

**Comment by vext01** [comment]
splat.

**Pull request merged**